ext_emconf.php

```diff
@@ -13,7 +13,8 @@
     'version' => '1.1.5',
     'constraints' => [
         'depends' => [
-            'typo3' => '9.0.0-11.5.99',
+            'php' => '8.0.0-8.4.99',
```

Comment:
TYPO3 v12 requires PHP 8.1, not PHP 8.0 (see https://typo3.org/cms/roadmap#section-php-compatibility-chart and https://get.typo3.org/version/12#system-requirements).

Reply:
This extension still allows TYPO3 10.

Reply:
technically TYPO3 v10 does not work with PHP 8, but I assume this will be resolved by composer then, even though it does not make sense in this case.

Reply:
Now I have changed it to require TYPO3 11.5 as minimum.